### PR TITLE
RC-97 display spinner while loading

### PR DESCRIPTION
**Pull request by GrimStal**
### **Jira ticket**

[RC-97](https://dbmedialab.atlassian.net/browse/RC-97)

### **What is the problem?**

Want to see a spinner while loading. It have to be able to implement into any page and preferable to be written on scss, not an image

### **How the problem was fixed**

- Added an npm package with a few different spinners
- Added import of scss styles
- Fixed some of them by our needs

### **What have been changed additionally**

- Nothing

### **Testing done**

Manually

### CODE REVIEW

File: src/front/scss/layout/_spinner.scss [hunk 1]
```diff
@@ -0,0 +1,11 @@
+@import '../../../../node_modules/spinkit/scss/spinners/3-wave';
```
Comment (liordoe) [inline src/front/scss/layout/_spinner.scss]: does absolute path import not configured in this project?

File: src/front/component/ModalWindow.tsx [hunk 2]
```diff
@@ -89,6 +90,9 @@ const AlertIcon : React.StatelessComponent <IAlertIcon> = (props : IAlertIcon) =
 	return (<svg height={height} version="1.1" viewBox="0 0 16 16" width={width} xmlns="http://www.w3.org/2000/svg"><title/><defs/><g fill="none" fillRule="evenodd" id="Icons with numbers" stroke="none" strokeWidth="1"><g fill={color} id="Group" transform="translate(-96.000000, -432.000000)"><path d="M103,443 L103,445 L105,445 L105,443 Z M104,448 C99.5817218,448 96,444.418278 96,440 C96,435.581722 99.5817218,432 104,432 C108.418278,432 112,435.581722 112,440 C112,444.418278 108.418278,448 104,448 Z M103,435 L103,442 L105,442 L105,435 Z M103,435" id="Oval 208 copy"/></g></g></svg>);
 };
 
+const { messages, locale } = (window['app'] && window['app'].localization) ||
```
Comment (liordoe) [inline src/front/component/ModalWindow.tsx]: :+1:

Comment (GrimStal) [reply src/front/component/ModalWindow.tsx]: :+1:

File: src/front/scss/layout/_spinner.scss (hunk 1 above)
Comment (GrimStal) [reply src/front/scss/layout/_spinner.scss]: seems like doesn't want to do it in pre-processor

Comment (DonikaV) [reply src/front/scss/layout/_spinner.scss]: Did you tried this?
```css
@import '~foundation-sites/scss/spinners/3-wave';

Comment (GrimStal) [reply src/front/scss/layout/_spinner.scss]: No. But I'll try

**Pull request merged**